logdevice/common/test/DataSizeRequestTest.cpp

```diff
@@ -378,12 +388,16 @@ TEST(DataSizeRequestTest, DeadEnd2) {
   ASSERT_TRUE(req.haveDeadEnd());
   ASSERT_FALSE(req.isMockGracePeriodTimerActive());
   // Answer should be (SUM / 4 responses) * 8 nodes = ~18.85 GB.
+  
+  // WHY 4 responses ???
+  
```

Comment:
Yeah, the comment is probably misleading. The fourth one is not a response. We got 3 E::OK, and one node (N1) is AUTH_EMPTY which is assumed to be 0.
So the average per node is (10122122122 + 10100100100 + 8055055055 + 0)/4 = 7.06 GB per node.
The estimated total size of the data is 7.06 * 8 nodes = 56.48 GB per node.
The data is replicated on 3 nodes, so the amount of actual data is 56.48 / 3 = ~18.82 GB.
That's my interpretation to this calculation :)

Reply:
Thanks for the clarification